static/js/components/VideoPlayer.js

```diff
@@ -237,9 +236,6 @@ export default class VideoPlayer extends React.Component<*, void> {
 
   imageExists(url: string, callback: Function) {
     const img = new Image()
-    img.onload = function() {
-      callback(true)
-    }
     img.onerror = function() {
       callback(false)
```

Comment:
the callback we're passing in (we're only using this to call `switchVideoSource`) doesn't take a boolean as an argument anymore, so I think you could do either

```js
img.onerror = () => callback()
```

or maybe just directly call

```js
img.onerror = this.switchVideoSource
```

and not bother having this function (`imageExists`) take the callback argument

Reply:
✅